.github/workflows/ci.yml

```diff
@@ -13,30 +13,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', 'pypy-3.9']
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-3.9']
 
     steps:
     - uses: actions/checkout@v4
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
-    - uses: actions/cache@v4
-      with:
-        path: ~/.cache/pip
-        key: ${{ runner.os }}-python-${{ matrix.python-version }}-pip-${{ hashFiles('.github/workflows/ci.yml') }}
-        restore-keys: ${{ runner.os }}-python-${{ matrix.python-version }}-pip
+        cache: 'pip'
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
         python -m pip install tox tox-gh-actions
         sudo apt-get install libmemcached-dev
     - name: Lint
-      if: matrix.python-version == '3.10'
+      if: matrix.python-version == '3.13'
```

Comment:
Please try running tox on your local machine.

This will require adding `setuptools` to `lint-requirements.txt` and dropping Python < 3.9 from `pyproject.toml`
* #608

Reply:
This also appears to introduce a formatting difference in `pymemcache/client/base.py`.

Reply:
I just pushed a commit moving that back to `3.10` to get the tests passing